.github/workflows/benchmark-suite.yml

```
@@ -0,0 +1,185 @@
+      - completed
+
+  # Manual trigger with options
+  workflow_dispatch:
+    inputs:
+      scenario:
+        description: 'Specific scenario to run (leave empty for all)'
+        required: false
+        default: ''
+        type: choice
+        options:
+          - ''
```

actionlint (1.7.10)

string should not be empty

(syntax-check)